index.html

```diff
@@ -33,23 +33,23 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <!-- HTML Meta Tags -->
-    <title>Push App (Previously EPNS) | Communication Protocol of Web3</title>
-    <meta name="description" content="Push App (Previously EPNS) | Communication Protocol of Web3">
+    <title>Push Alpha App (Previously EPNS) | Communication Protocol of Web3</title>
+    <meta name="description" content="Push Alpha App (Previously EPNS) | Communication Protocol of Web3">
 
     <!-- Facebook Meta Tags -->
-    <meta property="og:url" content="https://app.push.org">
+    <meta property="og:url" content="https://alpha.push.org">
```

Comment:
@HarshRajat I think you committed the changes in index.html, robots.txt and sitemap.txt after alpha deployment.

Reply:
Since the deployment script takes care of the meta property, committing index.html, robots and sitemap can be ignored?

Reply:
Yeah that right. But still changing it like this might be confusing, since we already got a index.html files.

Reply:
agreed